.test-infra/jenkins/CommonJobProperties.groovy

```diff
@@ -322,6 +322,9 @@ class CommonJobProperties {
         // Install job requirements for Python SDK.
```

Comment:
can you also add `pip install --upgrade tox==3.0.0 grpcio-tools==1.3.5` for Python SDK? This shows in Python gradle `setupVirtualenv` ([here](https://github.com/apache/beam/blob/master/sdks/python/build.gradle#L38)) which is used to run all Pre/Post-commit.